ee/vellum_ee/workflows/display/workflows/base_workflow_display.py

```diff
@@ -72,6 +72,7 @@
 class WorkflowSerializationResult(UniversalBaseModel):
     exec_config: Dict[str, Any]
     errors: List[str]
+    dataset: Optional[List[Dict[str, Any]]] = None
```

Comment:
each element in this list should look like this: `{"label":str, "inputs": {}}`. But we don't need to encode it here, just inform what we append below